web-common/src/features/dashboards/dashboard-fetch-mocks.ts

```diff
@@ -4,7 +4,6 @@ import type {
   V1MetricsViewSpec,
   V1TimeRangeSummary,
 } from "@rilldata/web-common/runtime-client";
-import { wait } from "@testing-library/user-event/dist/utils";
```

Comment:
Interesting. I removed the import because I looked through the `@testing-library/user-event` documentation and didn't see `wait` as a documented function.